src/foraging_gui/TransferToNWB.py

```diff
@@ -171,6 +180,12 @@ class obj:
         # Performance
         'foraging_efficiency': _get_field(obj, 'B_for_eff_optimal'),
         'foraging_efficiency_with_actual_random_seed': _get_field(obj, 'B_for_eff_optimal_random_seed'),
+
+        # Optogenetics
+        'laser_1_calibration_power': float(_get_field(obj.Opto_dialog, 'laser_1_calibration_power')),
+        'laser_2_calibration_power': float(_get_field(obj.Opto_dialog, 'laser_2_calibration_power')),
+        'laser_1_target_areas': _get_field(obj.Opto_dialog, 'laser_1_target',default='None') or 'None',
+        'laser_2_target_areas': _get_field(obj.Opto_dialog, 'laser_2_target',default='None') or 'None',
```

Comment:
Yon don't need `or 'None'` if your default value is 'None' because it will not take effect.

Reply:
Updated